Tests/SwiftGRPCTests/ClientTestExample.swift

```diff
@@ -92,6 +92,32 @@ extension ClientTestExample {
     XCTAssertEqual([Echo_EchoRequest(text: "foo")], fakeService.getRequests)
   }
   
+  func testClientAsynchronous() throws {
+    let fakeService = Echo_EchoServiceTestStub()
+    fakeService.getResponses.append(Echo_EchoResponse(text: "bar"))
+    
+    let client = ClientUnderTest(service: fakeService)
+    
+    let completionHandlerExpectation = expectation(description: "request completion handler called")
+
+    _ = try client.service.get(Echo_EchoRequest(text: "foo")){ (response:Echo_EchoResponse?, callResult:CallResult) in
```

Comment:
Still missing spaces:

`try client.service.get(Echo_EchoRequest(text: "foo")) { (response, callResult) in`

Please try to adhere to how the rest of the project uses whitespace.

Reply:
Still not resolved.